pkgs/by-name/wa/watchdogd/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook, libite, libuev
+, libconfuse }:
+stdenv.mkDerivation rec {
+  pname = "watchdogd";
+  version = "3.5";
+
+  src = fetchFromGitHub {
+    owner = "troglobit";
+    repo = "watchdogd";
+    rev = "${version}";
```

Unnecessary string interpolation

```suggestion
    rev = version;
```